docs/install/tutorials/rpi.rst

```diff
@@ -269,9 +269,14 @@ Installing Kolibri
       sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
       sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys DC5BAA93F9E4AE4F0411F97C74F88ADB3194DD81
       sudo apt update
-      sudo apt install kolibri
 
-   When asked questions during the installation, it is recommended that you use the default ``pi`` user for running Kolibri because it will have access to USB devices.
+#. Install ``kolibri`` and ``kolibri-server``:
+
+   .. code-block:: bash
+
+      sudo apt install kolibri kolibri-server
```

Comment:
kolibri-server is still only in the kolibri-proposed PPA, so it probably won't work as written here yet?

Reply:
No, I've intended that we move it to the stable PPA, and I don't think it's a big problem if we do that. We aren't announcing it widely, yet, but this would be an ideal start.